errbot/plugin_wizard.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import configparser
+import os
+import re
+import requests
+import sys
+
+from errbot import PY2
+
+if PY2:
+    input = raw_input
+
+
+def new_plugin_wizard(directory=None):
+    """
+    Start the wizard to create a new plugin in the current working directory.
+    """
+    if directory is None:
+        print("This wizard will create a new plugin for you in the current directory.")
+        directory = os.getcwd()
```

Comment:
This would complicate things a little and require loading the config before starting the wizard. It would also become more unpredictable, picking the current dir if `BOT_EXTRA_PLUGIN_DIR` is unspecified or that directory if it is specified.

I'd prefer to keep it as-is and require the user to pass in the directory to where they want it created if they don't want it in the current directory, which is more in line with how many other unix tools behave as well.